frontend/app/routes/public/apply/$id/child/home-address.tsx

```diff
@@ -0,0 +1,380 @@
+                required
+              />
+              <div className="mb-6 grid items-end gap-6 md:grid-cols-2">
+                <InputSanitizeField
+                  id="home-city"
+                  name="city"
+                  className="w-full"
+                  label={t('apply-child:address.address-field.city')}
+                  maxLength={100}
+                  autoComplete="address-level2"
+                  defaultValue={defaultState.homeAddress?.city}
+                  errorMessage={errors?.city}
+                  required
+                />
+                <InputSanitizeField
+                  id="home-postal-code"
+                  name="postalZipCode"
+                  className="w-full"
+                  label={isPostalCodeRequired ? t('apply-child:address.address-field.postal-code') : t('apply-child:address.address-field.postal-code-optional')}
+                  maxLength={100}
+                  autoComplete="postal-code"
+                  defaultValue={defaultState.homeAddress?.postalCode ?? ''}
+                  errorMessage={errors?.postalZipCode}
+                  required={isPostalCodeRequired}
+                />
+              </div>
+              {homeRegions.length > 0 && (
+                <InputSelect
+                  id="home-province"
+                  name="provinceStateId"
+                  className="w-full sm:w-1/2"
+                  label={t('apply-child:address.address-field.province')}
+                  defaultValue={defaultState.homeAddress?.province}
+                  errorMessage={errors?.provinceStateId}
+                  options={[dummyOption, ...homeRegions]}
+                  required
+                />
+              )}
+              <InputSelect
+                id="home-country"
+                name="countryId"
+                className="w-full sm:w-1/2"
+                label={t('apply-child:address.address-field.country')}
+                autoComplete="country"
+                defaultValue={defaultState.homeAddress?.country ?? ''}
+                errorMessage={errors?.countryId}
+                options={countries}
+                onChange={homeCountryChangeHandler}
+                required
+              />
+            </div>
+          </fieldset>
+          {editMode ? (
+            <div className="flex flex-wrap items-center gap-3">
+              <Dialog open={addressDialogContent !== null} onOpenChange={onDialogOpenChangeHandler}>
+                <DialogTrigger asChild>
+                  <LoadingButton
+                    aria-expanded={undefined}
+                    variant="primary"
+                    id="save-button"
+                    type="submit"
+                    name="_action"
+                    value={FORM_ACTION.submit}
+                    loading={isSubmitting}
+                    data-gc-analytics-customclick="ESDC-EDSC:CDCP Apply Application Form-Adult:Save - Home address click"
```

Comment:
```suggestion
                    data-gc-analytics-customclick="ESDC-EDSC:CDCP Apply Application Form-Child:Save - Home address click"
```

Same comment for the rest of the `data-gc-analytics-customclick` values in this route and `/mailing-address` for the child flow.